docs/bok/Activities/License-Management.md

```diff
@@ -0,0 +1,119 @@
+---
+title: License Compliance Management
+tags: 
+  - OSPO (Role)
+  - Developer (Role)
+  - Legal Risk
+---
+
+There are several key points that a large enterprise should consider to ensure compliance with open-source license obligations:
+
+1. License identification: The first step is to identify the open-source licenses used in your products and understand the terms and conditions associated with each license.
+2. License compatibility: It is important to ensure that the licenses of the open-source components used in your products are compatible with each other and with your own licenses.
+3. License obligations: Each open-source license has its own obligations, such as the requirement to distribute source code, include a copyright notice, or provide a copy of the license.
+4. Legal review: It is important to have a qualified legal professional review the open-source licenses and obligations to ensure compliance.
+5. Employee education: The enterprise should educate its employees on the importance of compliance with open-source licenses and the obligations associated with each license.
+6. License distribution: The enterprise should ensure that the open-source licenses and relevant notices are included in the distributions of their products.
+
+## License Identification
+
+### For Individual Software Projects
+
+Individual software projects often include direct and transitive dependencies on other pieces of open source software.  There are plenty of tools for cataloguing licenses being used:
+
+1. [Scancode](https://github.com/nexB/scancode-toolkit), recommended here: https://softwarerecs.stackexchange.com/questions/40068/tools-to-scan-code-for-open-source-licenses-and-copyrights
+
+2. [License Checker](https://www.npmjs.com/package/license-checker) which is an NPM module you can use in node projects to list the licenses used by the project.
```

Comment:
Can we add [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort) here? This is the tool developed by mostly German automotive OSPOs under the Linux Foundation.

Reply:
HI @tsteenbe - is the ORT more suitable for the whole-estate section? Seems like this is a great open-source option to add in there.  I might break those out into a list too.